about.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About IBS</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body class="container-fluid">
+    <section class="header bg-header">
+        <header class="container-fluid bg-navbar bg-white shadow p-1 mb-3 bg-body rounded">
+            <div class="top-menu">
+                <ul class="top-menu-wrap">
+                  <li class="menu2"><a><img class="facebook-page" src="./images/iconfacebook.png" alt="facebook"></a></li>
+                  <li class="menu2"><a><img class="twitter-page" src="./images/icontwitter.png" alt="twitter"></a></li>
+                  <li class="menu2"><a>English</a></li>
+                  <li class="menu2"><a>My Page</a></li>
+                  <li class="menu2"><a>Logout</a></li>
+                </ul>
+              </div>
+            <div class="container">
+              <nav class="navbar navbar-expand-lg navbar-light">           
+                <div class="container-fluid">
+                  <a class="navbar-brand d-none d-sm-block" href="/index.html"><img class="logo" src="./images/logo1.jpg" alt="logo"></a>
+                  <div class="custom-toggler">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                  </div>
+                  <div class="collapse navbar-collapse d-none d-lg-block" id="navbarDesktop">
+                    <ul class="navbar-nav ms-auto mb-2 mb-lg-0 mainmenu">
+                        <li>
+                            <a class="nav-link active red text-link" aria-current="page" href="/about.html">About</a>
+                          </li>
+                          <li>
+                            <a class="nav-link" href="#program">Program</a>
+                          </li>
```

Comment:
Kindly ensure the font properties of your nav links are consistent with the design🥲 as are required to match the design in this milestone. The font property of your program link looks different from the others.

<img width="956" alt="nav" src="https://user-images.githubusercontent.com/64643995/233152596-f8479ccd-7e01-423e-a46a-865e606f46ff.png">